system/Autoloader/FileLocator.php

```diff
@@ -145,10 +145,10 @@ public function getClassname(string $file): string
 
             if ((isset($tokens[$i - 2][1]) && ($tokens[$i - 2][1] === 'phpnamespace' || $tokens[$i - 2][1] === 'namespace')) || ($dlm && $tokens[$i - 1][0] === T_NS_SEPARATOR && $token[0] === T_STRING)) {
                 if (! $dlm) {
-                    $namespace = 0;
+                    $namespace = '';
```

Comment:
I don't know why the original code is `$namespace = 0;`.
If `$namespace` is set to `0`, is the case is just a bug?

Please make another PR to fix this issue.

Reply:
After changing 151 line in this file it's failing:
1) CodeIgniter\Autoloader\FileLocatorCachedTest::testGetClassNameFromClassFile
Failed asserting that two strings are identical.
--- Expected
+++ Actual
@@ @@
-'CodeIgniter\Autoloader\FileLocatorTest'
+'0\CodeIgniter\Autoloader\FileLocatorTest'